gallery/src/components/NxTransferListHalf/NxTransferListHalfPage.tsx

```diff
@@ -108,6 +108,15 @@ const NxTransferListPage = () =>
                 the <NxCode>isSelected</NxCode> prop) and the id of the item.
               </NxTable.Cell>
             </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>disableItemMove</NxCode></NxTable.Cell>
+              <NxTable.Cell>boolean</NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell></NxTable.Cell>
+              <NxTable.Cell>
+                Set to true to disable moving item feature and hide the move icon on each item.
```

Comment:
Maybe something like : 'When true it disable’s the ability to transfer or remove items by hiding the item’s icon.'